packages/es-components/src/components/controls/buttons/Button.js

```diff
@@ -10,7 +10,12 @@ const StyledButton = styled.button`
   background-color: ${props => props.colors.bgColor};
   border: 2px solid transparent;
   border-color: ${props => props.colors.bgColor};
-  border-radius: ${props => props.buttonSize.borderRadius};
+  border-bottom-left-radius: ${props => props.buttonSize.borderRadius};
+  border-bottom-right-radius: ${props =>
+    props.split ? 0 : props.buttonSize.borderRadius};
+  border-top-left-radius: ${props => props.buttonSize.borderRadius};
+  border-top-right-radius: ${props =>
+    props.split ? 0 : props.buttonSize.borderRadius};
```

Comment:
Would it make more sense to pass in `leftRadius` and `rightRadius` props from outside, rather than having these ternaries here?